### ISSUE TITLE
Appearance and collections panel becomes unresponsive with Auctionator installed.

### ISSUE THREAD

**Issue by DeeRez** [nezroy/DeModal#3]
With Auctionator installed the appearance and collection window loses text and becomes non-interactive.

![image](https://user-images.githubusercontent.com/30647070/172440015-a5d044a6-635c-4639-a9fc-4f8704e821fd.png)

Disabling Auctionator makes the window behave properly.

**Comment by nezroy** [comment]
The collections panel is weird. I will tackle the weirdness.

**Comment by Salmanius** [comment]
I have this issue too, but even when Auctionator is disabled. I found the culprit to be Rarity.

**Comment by nezroy** [comment]
I don't think this is related to auctionator or any other addons. I believe this is a duplicate of #1. Closing as duplicate.
